README.md

```diff
@@ -15,16 +15,21 @@
 APILyzer: A simple API Analyzer
 ===============================
 
-View this plugin's documentation at: http://revelc.github.io/apilyzer-maven-plugin
+[APILyzer][2] is a maven plugin that detects illegal changes to a maven
```

Comment:
Could move most of this documentation to website and have just the 1st sentence in readme